### PR TITLE
flake8 fixes

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
- fix flake8
- no longer install/test in place

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

## Testing
Existing tests

## Checklist
_Put an `x` in the boxes that apply._

- [x] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [x] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: .github/test_real.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 set -e
+
+cd tests
```
Comment (akleb) [inline .github/test_real.sh:4]: Is this the fix to allow it to be installed without link? I thought testflo worked from the repo directory?

Comment (nwu63) [reply .github/test_real.sh:4]: It's a bit more complicated than that, see the explanation in mdolab/pygeo#99.

**Review by akleb — APPROVED** [review]

File: pyspline/pyCurve.py [hunk 2]
```diff
@@ -894,7 +894,7 @@ def writeIGES_parameters(self, handle, Pcount, counter):
                 counter += 1
                 pos_counter = 0
 
-        for i in range(self.nCtl):
+        for _i in range(self.nCtl):
```
Comment (bernardopacini) [inline pyspline/pyCurve.py:897]: This is minor, but what is the motivation for this indexing change?

Comment (nwu63) [reply pyspline/pyCurve.py:897]: Flake8 checks for unused loop variables, and says they should be preceded by an underscore to show that it's intentionally unused.

**Review by bernardopacini — APPROVED** [review]

**Pull request merged**